main.go

```diff
@@ -121,9 +127,19 @@ func getK8sManagerOptions(exiter applicationExiter) manager.Options {
 	return options
 }
 
-func configureLogger() {
+func configureLogger(exiter applicationExiter) {
+	logMode := false
+	logModeEnv, err := getEnvVar(logModeEnvVar)
+
+	if err == nil {
+		logMode, err = strconv.ParseBool(logModeEnv)
+		if err != nil {
+			exiter.Exit(err)
```

Comment:
If the user set something other than `true` or `false` into `LOG_MODE` (e.g. "DEBUG"), is this error message enough to make the user understand what he/she did wrong?